### ISSUE TITLE
CASA 6.x installation

### ISSUE THREAD

**Issue by drtobybrown** [canfar/arcades#36]
Since CASA versions >=6.0 can be called from within standard python scripts, we need a method for the user to specify which version of casa they want to call.

We can test how best to do this with CASA 6.1 but if it's easy then what about `casa610`?

**Comment by drtobybrown** [comment]
The pip wheel install of casa 6.1 requires python 3.6 so we’ll need to have that kernel for a demo.

**Comment by drtobybrown** [comment]
Possible solution could be combined with #44

**Comment by brianmajor** [comment]
Yes, possibly a new kernel, or, a new casa focussed notebook-server with casa 6 and other required software pre-installed.
